docs/Validation-and-Serialization.md

```diff
@@ -163,8 +163,11 @@ const bodyJsonSchema = {
 }
 
 const queryStringJsonSchema = {
-  name: { type: 'string' },
-  excitement: { type: 'integer' }
+  type: 'object',
+  properties: {
+    name: { type: 'string' },
+    excitement: { type: 'integer' }
+  }
 }
 
 const paramsJsonSchema = {
```

Comment:
@danechitoaie would you mind updating this schema as well?

Reply:
done